.gitlab-ci.yml

```diff
@@ -344,112 +344,17 @@ Image Tests:
           - aws/rhel-9.2-nightly-aarch64
         INTERNAL_NETWORK: ["true"]
 
-OSTree:
+Trigger-rhel-edge-ci:
   stage: test
-  extends: .terraform/openstack
-  rules:
-    - !reference [.upstream_rules_all, rules]
-  script:
-    - schutzbot/deploy.sh
-    - /usr/libexec/tests/osbuild-composer/ostree.sh
-  parallel:
-    matrix:
-      - RUNNER:
-          - rhos-01/fedora-36-x86_64
-          - rhos-01/fedora-37-x86_64
-          - rhos-01/rhel-8.4-ga-x86_64
-          - rhos-01/rhel-8.7-ga-x86_64
-          - rhos-01/rhel-9.1-ga-x86_64
-          - rhos-01/rhel-8.8-nightly-x86_64
-          - rhos-01/rhel-9.2-nightly-x86_64
-          - rhos-01/centos-stream-8-x86_64
-          - rhos-01/centos-stream-9-x86_64
-
-New OSTree:
-  stage: test
-  extends: OSTree
-  script:
-    - schutzbot/deploy.sh
-    - /usr/libexec/tests/osbuild-composer/ostree-ng.sh
-  parallel:
-    matrix:
-      - RUNNER:
-          - rhos-01/fedora-36-x86_64
-          - rhos-01/fedora-37-x86_64
-          - rhos-01/rhel-8.8-nightly-x86_64
-          - rhos-01/rhel-9.2-nightly-x86_64
-          - rhos-01/centos-stream-8-x86_64
-          - rhos-01/centos-stream-9-x86_64
-
-OSTree simplified installer:
-  stage: test
-  extends: OSTree
-  script:
-    - schutzbot/deploy.sh
-    - /usr/libexec/tests/osbuild-composer/ostree-simplified-installer.sh
-  parallel:
-    matrix:
-      - RUNNER:
-          - rhos-01/rhel-8.8-nightly-x86_64
-          - rhos-01/centos-stream-8-x86_64
-          - rhos-01/rhel-9.2-nightly-x86_64
-          - rhos-01/centos-stream-9-x86_64
-
-OSTree raw image:
-  stage: test
-  extends: OSTree
+  extends: .terraform
   rules:
     - !reference [.upstream_rules_all, rules]
   script:
-    - schutzbot/deploy.sh
-    - /usr/libexec/tests/osbuild-composer/ostree-raw-image.sh
+    - 'curl -u "${SCHUTZBOT_LOGIN}" -X POST -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/osbuild/rhel-edge-ci/dispatches -d "{\"event_type\":\"osbuild-composer-ci\",\"client_payload\":{\"pr_number\":\"${CI_COMMIT_BRANCH}\"}}"'
   parallel:
```

Comment:
IMO it would be great if this trigger executes inside GitHub actions, similar to `.github/workflows/trigger-gitlab.yml`. That will reduce the turnaround time and also reduce the need of deploying 1 VM in AWS only to execute curl. 

We can handle this in a separate PR if we don't want to delay this work any longer.

Reply:
The idea here is to trigger the tests only after the rpmbuild stage has passed to make sure the rpms are available for testing... But your point is valid that we don't have to spin up a whole VM, we can just use the `bash` executor similar to how `init` and `finish` stages are written. That will basically just execute the curl on the main CI scheduler machine without spinning up anything.